pkg/gofr/gofr.go

```diff
@@ -39,12 +39,12 @@ type App struct {
 
 // RegisterService adds a grpc service to the gofr application.
 func (a *App) RegisterService(desc *grpc.ServiceDesc, impl interface{}) {
-	a.grpcRegistered = true
 	a.container.Logger.Infof("Registering GRPC Server: %s", desc.ServiceName)
 	a.grpcServer.server.RegisterService(desc, impl)
+	a.grpcRegistered = true
```

Comment:
why this change?

Reply:
was suggested, since we should set it to be true after it is actually registered!